drivers/fpga/xrt/lib/lib-drv.c

```diff
@@ -123,13 +122,59 @@ void xrt_unregister_driver(struct xrt_driver *drv)
 }
 EXPORT_SYMBOL_GPL(xrt_unregister_driver);
 
-static int __find_driver(struct device_driver *drv, void *_data)
+static int __get_driver(struct device_driver *drv, void *_data)
 {
 	struct xrt_driver *xdrv = to_xrt_drv(drv);
 	struct xrt_find_drv_data *data = _data;
 
 	if (xdrv->subdev_id == data->id) {
-		data->xdrv = xdrv;
+		if (xdrv->driver.owner && xdrv->driver.owner != THIS_MODULE) {
+			if (try_module_get(xdrv->driver.owner))
+				*(int *)data->arg = 0;
```

Comment:
if try_module_get() failed, do we want to return some error code through data->arg?